gidgethub/abc.py

```diff
@@ -16,9 +16,11 @@ class GitHubAPI(abc.ABC):
     """Provide an idiomatic API for making calls to GitHub's API."""
 
     def __init__(self, requester: str, *, oauth_token: Opt[str] = None,
+                 jwt: Opt[str] = None,
```

Comment:
Rethinking this (out loud). The json web token expires after max 10 minutes, so maybe it doesn't make sense to be initialized here.
I'm kinda wanting to store the private key instead. 🤔

Reply:
I decided to not add `jwt` argument here because I just don't think it makes sense.